pkg/decode/decode_protobuf.go

```diff
@@ -149,6 +149,14 @@ func RecordToMap(fr *model.Record) config.GenericMap {
 
 	if len(fr.NetworkMonitorEventsMD) != 0 {
 		out["NetworkEvents"] = fr.NetworkMonitorEventsMD
+		for _, event := range fr.NetworkMonitorEventsMD {
+			// override drop fields when network event action is dropped
+			if event["Action"] == "drop" {
+				out["PktDropBytes"] = fr.Metrics.Bytes
```

Comment:
This is not right the same flow could be going through allow ingress and drop egress events ?

Reply:
Could we have the amount of packets / bytes from the event ?

Previously we had these amounts of drops. This is displayed in the plugin + dashboards so it would be very useful to get that back somehow

Reply:
its not a single event same flow can have allow and deny event or 2 allow and 1 deny or any other combo so hard to tell, the drop we had will still work for everything else except netpol and adminnetpol right ?

Reply:
Yes of course but the goal here is to be able to make graphs with the amount of bytes / packets / flows denied because of policies.

Reply:
That's how @jotak managed it for now: https://github.com/netobserv/network-observability-operator/pull/863 + https://github.com/netobserv/flowlogs-pipeline/pull/739

The console plugin could rely on prometheus for these but will not be able to query loki without extra processing